joomla/joomla-cms
# Proxy JData to the Framework

#### Summary of Changes

The `JData` classes and the `joomla/data` Framework package are largely one in the same with the latter having received some new code not existing in the CMS.  Since this is basically duplicate code, let's merge sources.
#### Testing Instructions

If anyone's using these classes (sorry, core doesn't use them), make sure your code still works the same without changes; similar to `JRegistry`, the existing classes are aliased to their namespaced counterparts.  The only gotcha might be if you're using Reflection on the private class member variables, but as those are private anyway there's no room for a B/C break.  Otherwise, this is essentially a maintainer review thing.
#### Documentation Changes Required

N/A, the JData classes are not documented in the CMS workspace
